app/Forms/Components/ColorSelect.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Forms\Components;
+
+use Filament\Support\Colors\Color;
+use Filament\Forms\Components\Select;
+
+class ColorSelect extends Select
+{
+    protected int $shade = 600;
```

Comment:
Is 500 not typically the default?

Reply:
I believe I saw @danharrin was using 600 in the form builder so I wanted it to match. I don't think I've seen anywhere where 600 is explicitly the default we just picked 500 on a whim as far as I recall.

Reply:
Actually yeah I just doubled check and the forms use 600. If you set a primary color and check the hex you get a tailwind 600 value. For example I used Emerald which displays `#059669` which corresponds to Emerald 600 on the tailwind site.